CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 1.21.0
+
+This release includes bugfixes for the experimental CRI support, more stable integration tests, and some other interesting changes:
+
+- The `default-restricted` network changed from 172.16.28.0/24 to 172.17.0.0/26.
+- The detailed proposal for OCI support has been finalized.
+- Apps run with `--insecure-options=paths` can access `/proc/sys/` and `/sys/`.
```

Comment:
Unfortunately not yet, still blocked on https://github.com/coreos/systemd/pull/71#issuecomment-264225712. I suggest removing this line.